types/datetime.go

```diff
@@ -274,6 +274,11 @@ func (a Datetime) Milliseconds() int64 {
 	return a.value
 }
 
+// Time returns the time.Time representation of a Datetime.
+func (a Datetime) Time() time.Time {
+	return time.UnixMilli(a.value)
```

Comment:
Hmmmm, check on the Location for this `time.Time`. From a cursory reading of the code, I think it might be in the local timezone, which would be wrong.

Reply:
Enforcing UTC now, thanks!